PhotoPlusBackend/photoplus/src/main/java/pl/polsl/photoplus/model/entities/Image.java

```diff
@@ -0,0 +1,34 @@
+package pl.polsl.photoplus.model.entities;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import pl.polsl.photoplus.annotations.Patchable;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Lob;
+import javax.persistence.Table;
+
+@Entity(name = "images")
+@Table(name = "images")
+@Getter
+@Setter
+@NoArgsConstructor
+public class Image extends AbstractEntityModel {
+
+    public Image(final String name, final byte[] bytes) {
+        this.name = name;
+        this.bytes = bytes;
```

Comment:
You can try usage of @AllArgsConstructor. See [documentatnion](https://projectlombok.org/api/lombok/AllArgsConstructor.html)